components/checkbox.json

```diff
@@ -0,0 +1,52 @@
+{
+  "checkbox": {
```

Comment:
Can you add a "figma" property please linking to the relevant figma?

Reply:
Ah yes, done.